jobs.go

```diff
@@ -44,6 +44,7 @@ type JobsService struct {
 type Job struct {
 	Commit        *Commit    `json:"commit"`
 	CreatedAt     *time.Time `json:"created_at"`
+	Coverage      *float64   `json:"coverage"`
```

Comment:
Please do not use a pointer to builtin types. Looks fine otherwise 👍

Reply:
Ideally I'd like the API to not work this way, it's valid values are `null` and `float`. I've seen most API just go for the pointer approach -- here's some alternatives

```
type Coverage int
// And *Coverage
// or implement UnmarshalJSON on the type instead and ditch the pointer altogether 
```

LMK, otherwise I'll check for an example elsewhere in the code. I'll have have an update in the next few hours